src/is_face_detector/utils.py

```diff
@@ -0,0 +1,42 @@
+import json
+from is_wire.core import Logger
+import argparse
+import numpy as np
+import cv2
+from is_msgs.image_pb2 import Image, ObjectAnnotations, ObjectLabels
+from google.protobuf.json_format import Parse
+from options_pb2 import FaceDetectorOptions
+
+def load_options():
```

Comment:
avoid to use argparse inside a function to load options. This function must only load the options from a given options file!